java-frontend/src/main/java/org/sonar/java/bytecode/cfg/BytecodeCFGBuilder.java

```diff
@@ -92,7 +92,8 @@ public MethodVisitor visitMethod(int access, String name, String desc, String si
   }
 
   public static class BytecodeCFG {
-    List<Block> blocks;
+    @VisibleForTesting
+    public List<Block> blocks;
```

Comment:
I am not sure I understand why this modification was actually required ?

Reply:
leftover of some intermediate change, seems no longer necessary, reverted